crates/aquascope_serve/src/main.rs

```diff
@@ -96,10 +96,8 @@ impl From<container::ServerResponse> for ServerResponse {
 pub enum Error {
```

Comment:
Should we also use this as an opportunity to get rid of the double request types higher up in this file? I don't think we need both?